### ISSUE TITLE
Disable github action

### ISSUE THREAD

**Issue by Luvata** [Jekyll-Garden/jekyll-garden.github.io#26]
After delete the repo (e.g luvata.github.io) that was created from this template, whenever I create a new repo with the same name, even if my repo have only one github workflows (defined in `.github/workflows/publish.yaml`)  there is always a github action named `pages-build-deployment` that run this template. Sorry I'm quite new to github action, how can I disable this github action?

Test project: https://github.com/Luvata/luvata.github.io/tree/master/.github/workflows

**Comment by hfactor** [comment]
I think it's a GitHub pages feature. Check [Github Changelog](https://github.blog/changelog/2021-12-16-github-pages-using-github-actions-for-builds-and-deployments-for-public-repositories/), [Your test projects' Workflows](https://github.com/Luvata/luvata.github.io/actions/workflows/pages/pages-build-deployment) and [Github Pages Docs](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites)

I will test this from my side, but I don't think it has anything to with _this_ theme.

**Comment by Luvata** [comment]
I also don't think this issue could relate to your theme, but the static site that was generated by the `pages-build-deployment` have the same theme with jekyll garden, and it also happened after I used this template, so at some point it could be.

Oh I found the `github-pages` in "environment" in the setting, I tried to delete it but nothing change, the `pages-build-deployment` still run.

**Comment by Luvata** [comment]
I discovered that the template I used actually disable jekyll by default [publish](https://github.com/srid/neuron-template/blob/master/.github/workflows/publish.yaml#L14). With that, the `pages-build-deployment` will not override the custom build.

But I can only successfull deploy this gh-page by renaming my repo into e.g `blogs` instead of `my-gh-handle.github.io`. It's quite strange but it worked.

**Comment by Luvata** [comment]
I discovered a workaround by using the template as a repo with name different with the gh page (e.g blogs), then run the custom build once and after that rename the repo back to `[GH HANDLE].github.io`, then the custom build worked. 

So I guess it's a bug in gh page and not related to this theme. Sorry for bothering you.

Your theme looks sooo clean and the search page is really awesome, but I missed some functions from the custom build (e.g query, date time, ...) so I came back to my previous template. I will give it a go in the future release.

Once again, thank you for your time and your support. Have a nice weekends!
